### DISCUSSION TITLE
SLINT_BACKEND variable not working

### DISCUSSION THREAD

**Discussion by DennisZhangOiler** [slint-ui/slint#6831] [Q&A]
Running command with SLINT_BACKEND set to winit not wokring, slint still uses QT as backend.

**Comment by ogoffart** [comment]
What platform are you on?
How do you compile your application?
Do you see a warning in the console?
Are you sure the winit backend was enabled when building?

**Reply by DennisZhangOiler** [reply]
Hi I am on `5.15.153.1-microsoft-standard-WSL2+` with qt6 installed and I compile my app with either `SLINT_BACKEND=winit cargo r --release` or `export SLINT_BACKEND=winit \ cargo r --release`, I do see a warning but I don't think it's related to.
```
qt.qpa.plugin: Could not find the Qt platform plugin "wayland" in ""
QStandardPaths: wrong permissions on runtime directory /run/user/1000/, 0755 instead of 0700
```
I am pretty sure I set the environment variable correctly, but I am not sure that's not using winit.
